modules/security/pages/login-password.adoc

```diff
@@ -0,0 +1,22 @@
+= Complexity password policy and Login protection
+:description: Login, Password
+:pp: {plus}{plus}
+:page-aliases: README.adoc, readme.adoc
+
+Tigergraph provides complexity password policy and login protection to improve the security.
```

Comment:
```suggestion
TigerGraph provides a complex password policy and login protection to enhance security.
```

Reply:
@anjaliIthapeTG We don't have a complex policy (or at least I hope we don't). We offer optional built-in rules for the complexity of passwords, e.g. the password must have a letters, numbers, and special characters.

Reply:
@victorleeTG My apologies for the oversight. I’ll review the documentation mentioned on the website and will update again